storage/database/concepts.mdx

```diff
@@ -38,12 +38,20 @@ Allows you to create a full copy of your Database Instance (including users and
 
 ## Database backup
 
-A database backup is a dated export of a database stored on an offsite backup backend. A database can be restored from a backup.
+A database backup is a complete dated export of a Database Instance stored on an offsite backup backend. A database can be fully restored from a backup. 
 
 </Concept>
 
 <Concept opened>
 
+## Database snapshot
+
+A snapshot is a full copy of a **volume** of your Database Instance at a certain point in time. It allows you to recover your volume data, much like [backups](#Database-backup), but the snapshotted volume is deployed on a new Database Instance. Sanapshots by themselves are not backups, since they are only of particular volumes, but can be used as part of the backup process.
```

Comment:
```suggestion
A snapshot is a full copy of a **volume** of your Database Instance at a certain point in time. It allows you to recover your volume data, much like [backups](#database-backup), but the snapshotted volume is deployed on a new Database Instance. Snapshots by themselves are not backups, since they are only of particular volumes, but can be used as part of the backup process.
```